core/domain/services/graphql/connection_resolvers/PriceConnectionResolver.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\connection_resolvers;
+
+use EE_Ticket;
+use EE_Error;
+use EEM_Price;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+use InvalidArgumentException;
+use ReflectionException;
+
+/**
+ * Class PriceConnectionResolver
+ */
+class PriceConnectionResolver extends AbstractConnectionResolver
+{
+
+    /**
+     * @return EEM_Price
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
+    public function get_query()
```

Comment:
We can't do that because that method is expected by WP GraphQL in `WPGraphQL\Data\Connection\AbstractConnectionResolver`. All the connection resolvers extend that class.

Reply:
oh oops... sry, just realized what I wrote above is a bit wonky...

here's a better commnet:

similar to how `EventEspresso\core\domain\services\graphql\abstracts\AbstractRootQueryConnection` has a $model property and getter for it, you could do the same in `EventEspresso\core\domain\services\graphql\connection_resolvers\AbstractConnectionResolver` (which extends `WPGraphQL\Data\Connection\AbstractConnectionResolver` and then add:

```php
public function get_query()
{
	return $this->model;
}
```

to that class and then all of its children would have this method covered